samples/node/pub_sub_electron_node/README.md

```diff
@@ -62,64 +60,105 @@ Replace with the following with the data from your AWS account:
+Note that in a real application, you may want to avoid the use of wildcards in your ClientID or use them selectively. Please follow best practices when working with AWS on production applications using the SDK.
 
 </details>
 
 ## How to run
-
 ### Direct MQTT via mTLS
 
 To Run this sample using a direct MQTT5 connection with a key and certificate, go to the `node/pub_sub_electron_node` folder.
 1. Setup your credential. You need to fill in the credentials in the `node/pub_sub_electron_node/settings.ts` with your AWS endpoint, certificate file path, private key file path.
 
 2. Install node packages
 ``` sh
-npm install
+npm install .
 ```
 
-3. Build and Run
+3. Start Sample
 ```sh
-npm run build
 npm run start
 ```
 
-### Websockets
+### MQTT over Websockets with TLS
 
 To Run this sample using Websockets, go to the `node/pub_sub_electron_node` folder.
 1. Setup your credential. You will need to set your AWS credentials in your environment variables or local files. See the [authorizing direct AWS](https://docs.aws.amazon.com/iot/latest/developerguide/authorizing-direct-aws.html) page for documentation on how to get the AWS credentials, which then you can set to the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_SESSION_TOKEN` environment variables.
 
-2. Setup `settings.ts`. You will need setup the `region` and `endpoint` in `settings.ts` to setup the e
+2. Setup `node/pub_sub_electron_node/settings.ts`. You will need setup the `region` and `endpoint` in the setting file.
 
 3. Install node packages
 ```sh
-npm install
+npm install .
 ```
 
 4. Build and Run
 ```sh
-npm run build
 npm run start
 ```
 
-## Electron Q&A
-### Warning: `objc[79765]: Class WebSwapCGLLayer is implemented in both ` ?
+## Package
+Please refer to (Electron-tutorial-packaging)[https://www.electronjs.org/docs/latest/tutorial/tutorial-packaging] for packaging details.
+As our sample is using typescript, you would need include the compiled js files while packaging. You can config the `package.json` or `forge.config.js` to set it.
+Example `package.json`:
+```js
+"build": {
+    "files": [
+      "dist/*"
+    ]
+  }
+```
 
-This is an issue running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". The warning should not affect your development. The issue is fixed by Electron in v22. Unfortunately, our SDK currently only supports Electron@19 and below.
 
-More info: https://github.com/electron/electron/issues/33685
+## Electron Q&A
+### Warning: `objc[79765]: Class WebSwapCGLLayer is implemented in both ... `
+This is an issue running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". The warning should not affect your development. The issue is fixed by Electron in v22.
```

Comment:
Trivial: "This is an issue when running Electron on MacOS. The API has a name duplication for "WebSwapCGLLayer". This warning should not affect your development and has been fixed in Electron v22."

Note: Is it "WebSwapCGLLayer" or "WebSwapCGLayer". Just checking whether it's a double "L" in the name.